met-api/migrations/versions/c656f3f82334_add_languages_and_tenant_mapping.py

```diff
@@ -0,0 +1,788 @@
+      "iso_code": "si"
+    },
+    {
+      "language": "Slovak",
+      "iso_code": "sk"
+    },
+    {
+      "language": "Slovenian",
+      "iso_code": "sl"
+    },
+    {
+      "language": "Somali",
+      "iso_code": "so"
+    },
+    {
+      "language": "Southern Ndebele",
+      "iso_code": "nr"
+    },
+    {
+      "language": "Spanish",
+      "iso_code": "es"
+    },
+    {
+      "language": "Sundanese",
+      "iso_code": "su"
+    },
+    {
+      "language": "Swahili (Kiswahili)",
+      "iso_code": "sw"
+    },
+    {
+      "language": "Swati",
+      "iso_code": "ss"
+    },
+    {
+      "language": "Swedish",
+      "iso_code": "sv"
+    },
+    {
+      "language": "Tagalog",
+      "iso_code": "tl"
+    },
+    {
+      "language": "Tahitian",
+      "iso_code": "ty"
+    },
+    {
+      "language": "Tajik",
+      "iso_code": "tg"
+    },
+    {
+      "language": "Tamil",
+      "iso_code": "ta"
+    },
+    {
+      "language": "Tatar",
+      "iso_code": "tt"
+    },
+    {
+      "language": "Telugu",
+      "iso_code": "te"
+    },
+    {
+      "language": "Thai",
+      "iso_code": "th"
+    },
+    {
+      "language": "Tibetan",
+      "iso_code": "bo"
+    },
+    {
+      "language": "Tigrinya",
+      "iso_code": "ti"
+    },
+    {
+      "language": "Tonga",
+      "iso_code": "to"
+    },
+    {
+      "language": "Tsonga",
+      "iso_code": "ts"
+    },
+    {
+      "language": "Turkish",
+      "iso_code": "tr"
+    },
+    {
+      "language": "Turkmen",
+      "iso_code": "tk"
+    },
+    {
+      "language": "Twi",
+      "iso_code": "tw"
+    },
+    {
+      "language": "Uyghur",
+      "iso_code": "ug"
+    },
+    {
+      "language": "Ukrainian",
+      "iso_code": "uk"
+    },
+    {
+      "language": "Urdu",
+      "iso_code": "ur"
+    },
+    {
+      "language": "Uzbek",
+      "iso_code": "uz"
+    },
+    {
+      "language": "Venda",
+      "iso_code": "ve"
+    },
+    {
+      "language": "Vietnamese",
+      "iso_code": "vi"
+    },
+    {
+      "language": "Volapük",
+      "iso_code": "vo"
+    },
+    {
+      "language": "Wallon",
+      "iso_code": "wa"
+    },
+    {
+      "language": "Welsh",
+      "iso_code": "cy"
+    },
+    {
+      "language": "Wolof",
+      "iso_code": "wo"
+    },
+    {
+      "language": "Western Frisian",
+      "iso_code": "fy"
+    },
+    {
+      "language": "Xhosa",
+      "iso_code": "xh"
+    },
+    {
+      "language": "Yiddish",
+      "iso_code": "yi"
+    },
+    {
+      "language": "Yoruba",
+      "iso_code": "yo"
+    },
+    {
+      "language": "Zhuang, Chuang",
+      "iso_code": "za"
+    },
+    {
+      "language": "Zulu",
+      "iso_code": "zu"
+    }
+]
+
+def upgrade():
+    for index, value in enumerate(language_data):
+        op.execute("INSERT INTO language (created_date, updated_date, id, name, code) VALUES ('{0}', '{0}', '{1}', '{2}', '{3}')".format(sa.func.now(), index, value['language'], value['iso_code']))
```

Comment:
to avoid ID collisions, you may want to drop the existing languages in the table, unless this table is known to be empty